maven/bnd-export-plugin/src/test/resources/integration-test/export-test/pom.xml

```diff
@@ -0,0 +1,30 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>biz.aQute.bnd-test</groupId>
+    <artifactId>export-test</artifactId>
+    <version>3.2.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-export-maven-plugin</artifactId>
+                <configuration>
+                    <bndrun>
```

Comment:
This configuration seems different than normal multi-value things. I would have expected

```
<bndruns>
  <bndrun>test.bndrun</bndrun>
</bndruns>
```